libs/blocks/ost/ost.js

```diff
@@ -125,7 +125,7 @@ export async function loadOstEnv() {
     }
   }
 
-  ({ country, language } = getTacocatEnv(ENV_PROD, locale));
+  ({ country, language } = getTacocatEnv(true, locale));
```

Comment:
can we remove `env` parameter all together since it will be always `PRODUCTION`?

Reply:
not sure it will always be it: we need a way to test new wcs / aos / whatever commerce change. We need 
1. our codebase to be able to point to stage from time to time 
2. our codebase to be able to use draft landscape from time to time
1 & 2 will be handled later, now we are just not closing the door to 1.